tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayGradleWorkspaceProject.java

```diff
@@ -147,6 +152,25 @@ protected IStatus run(IProgressMonitor monitor) {
 
 		};
 
+		job.addJobChangeListener(
+			new JobChangeAdapter() {
+
+				@Override
+				public void scheduled(IJobChangeEvent event) {
+					Stream.of(
```

Comment:
Hey @simonjhy 

Please just move this into the job itself, just before we run the watch task.

Reply:
hi greg,  I changed it. Please check

Reply:
Can you resend?  thanks!